packages/edge/src/edge-headers.ts

```diff
@@ -68,6 +71,16 @@ function getHeader(request: Request, key: string): string | undefined {
   return request.headers.get(key) ?? undefined;
 }
 
+function getFlag(countryCode: string | undefined): string | undefined {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏add description

This function returns a string of unicode characters that represent the letters passed in, right? It took me a while to figure out what was happening here. Can you add a jsdoc description to this function?

For example:

```
'AA' -> '🇦🇦'
```